scripts/list-parser-links.lua

```diff
@@ -0,0 +1,15 @@
+-- Execute as `nvim --headless -c "luafile ./scripts/list-parser-links.lua"
```

Comment:
Nice!

Reply:
Not as clever as you might think. `lua ./scripts/list-parser-links.lua` would also work.